include/openthread/ip6.h

```diff
@@ -105,6 +117,7 @@ typedef struct otNetifAddress
 {
     otIp6Address           mAddress;                ///< The IPv6 unicast address.
     uint8_t                mPrefixLength;           ///< The Prefix length (in bits).
+    uint8_t                mAddressOrigin;          ///< The IPv6 address origin.
```

Comment:
```suggestion
    uint8_t                mAddressOrigin: 2;          ///< The IPv6 address origin.
```

Reply:
Would you change your suggestion to 
```suggestion
    uint8_t                mAddressOrigin: 2;       ///< The IPv6 address origin.
```
so that the alignment is correct and I can just apply it.

Reply:
nevermind. github allowed to modify the suggestion.